UnitsNet/UnitDefinitions/Mass.json

```diff
@@ -115,6 +115,34 @@
           "Abbreviations": [ "st" ]
         }
       ]
+    },
+    {
+      "SingularName": "ShortHundredweight",
+      "PluralName": "ShortHundredweight",
+      "FromUnitToBaseFunc": "x/0.022046226218487758",
+      "FromBaseToUnitFunc": "x*0.022046226218487758",
+      "XmlDocSummary": "The short hundredweight (abbreviation cwt) is a unit of mass equal to 100 pounds in US and Canada. In British English, the short hundredweight is referred to as the \"cental\".",
+      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Hundredweight",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cwt" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "LongHundredweight",
+      "PluralName": "LongHundredweight",
+      "FromUnitToBaseFunc": "x/0.01968413055222121",
+      "FromBaseToUnitFunc": "x*0.01968413055222121",
+      "XmlDocSummary": "The long or imperial hundredweight (abbreviation cwt) is a unit of mass equal to 112 pounds in US and Canada.",
+      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Hundredweight",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cwt" ]
```

Comment:
cwt is always the short hundredweight in US / Canada, so using `sh cwt` would be confusing here.  

Maybe someone from the UK or Australia can chime in on what `cwt` typically means (if anything).

I know the Hundredweight can't be used anymore for trade:

![image](https://cloud.githubusercontent.com/assets/5941336/24016411/5d022c1a-0a62-11e7-8b04-74ea3231dc93.png)
http://www.legislation.gov.uk/ukpga/1985/72/pdfs/ukpga_19850072_en.pdf